index.js

```diff
@@ -60,6 +60,17 @@ module.exports = {
 
         json.scripts = scripts(options);
 
+        if (options.packageManager === 'pnpm' || options.pnpm) {
+          delete json.workspaces;
+
+          json.pnpm = {
+            // TODO: update the blueprint's output to ESLint 8
```

Comment:
Without this prepare / tsc fail

eslint 8 is brought in via webpack, and tsc is just 🤷 when it comes to which version to use.
though, I don't know why these types are even relevant in the first place.

ultimately, the app blueprint will soon have eslint 8, and we can remove this.
Thanks, @bertdeblock for the update!